charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluentd-elasticsearch
-version: 2.2.1
+version: 2.2.2
```

Comment:
New features should raise minor version instead of bugfix version.
Please also fill out our pr template completely.